Networking/NetworkingTests/Requests/AuthenticatedRequestTests.swift

```diff
@@ -24,8 +24,13 @@ class AuthenticatedRequestTests: XCTestCase {
         let authenticated = AuthenticatedRequest(credentials: credentials, request: unauthenticatedRequest)
         let output = try! authenticated.asURLRequest()
 
+        guard case let .wpcom(username: _, authToken: authToken, siteAddress: _) = credentials else {
```

Comment:
Nit: this can be simplified without the labels for associated values:

```suggestion
        guard case let .wpcom(_, authToken, _) = credentials else {
```

Reply:
Done as part of https://github.com/woocommerce/woocommerce-ios/pull/8446/commits/b231bcef7029ee63fab0c1e723c9e645ed2e16e1